src/components/CreateVmWizard/steps/SummaryReview.js

```diff
@@ -74,10 +75,12 @@ ReviewBasic.propTypes = {
 }
 
 const ReviewNetworking = ({ id, vnicProfiles, network }) => {
+  const initialVnicNames = {}
+  initialVnicNames[EMPTY_VNIC_PROFILE_ID] = msg.nicNoVnicAssigned()
   const vnicNames = vnicProfiles.reduce((map, vnic) => {
     map[vnic.get('id')] = `${vnic.getIn(['network', 'name'])}/${vnic.get('name')}`
     return map
-  }, {})
+  }, initialVnicNames)
```

Comment:
Last nit-pick to avoid the `initialVnicNames`:

```suggestion
  }, {
    [EMPTY_VNIC_PROFILE_ID]: msg.nicNoVnicAssigned(),
  })
```

Reply:
Done.